app/assets/javascripts/dashboard/protocols.js.coffee

```diff
@@ -128,10 +128,15 @@ $(document).ready ->
           window.location = "/dashboard/protocols/#{protocol_id}/edit"
 
       $(document).on 'click', '.view-full-calendar-button', ->
-        protocol_id = $(this).data('protocol-id')
+        protocol_id = $(this).data('protocolId')
+        statuses_hidden = $(this).data('statusesHidden')
```

Comment:
`ServiceCalendarsController#view_full_calendar` now takes a param `statuses_hidden` to choose which SSR's to hide based on status. So does `Dashboard::ProtocolsController#show` when rendering the spreadsheet.